scripts/adb_shell.sh

```diff
@@ -1,10 +0,0 @@
-#!/usr/bin/expect --
-# For licensing see accompanying LICENSE file.
```

Comment:
any specific reason to delete this file? 

is it not helpful for spawning to device?

Reply:
Yes, there are some reasons:
1. there might be more than 1 device(s) to connect to. I think it's up to the developer which device to be used. 
2. I haven't seen any project providing automatic adb connection. we can document how to connect and run CLI binary, but the actual execution is on developers. 
3. instead, we provide top level "make test' command which does adb connection, run whisperax_cli, and finally shows the transcription result.